src/components/Label/DisposableLabel.test.js

```diff
@@ -1,18 +1,17 @@
 import React from 'react';
-import { mount } from 'enzyme';
-import { DisposableLabel } from './index';
-import { MockLabelRemove } from './__mocks__/mockLabelExamples';
+import { shallow } from 'enzyme';
+import DisposableLabel from './DisposableLabel';
+import Label from './Label';
+import { noop } from '../../common/helpers';
 
-test('Label renders properly', () => {
-  const component = mount(
-    <DisposableLabel type="default">Some text</DisposableLabel>
-  );
-  expect(component.render()).toMatchSnapshot();
+test('defaults props', () => {
+  const view = shallow(<DisposableLabel />);
+  expect(view).toMatchSnapshot();
```

Comment:
do you want to start using names?
i.e. `toMatchSnapshot('default props snapshot'));` ? or fine w/ leaving those out? sorry was confused and start using them ;)

Reply:
It definitely does not hurt.  The real benefit comes from when there are multiple snapshots in 1 test.  By default, they will get the test's name.  If there are multiple you just get a 1 2 3 etc. I think it would be a good idea though if we are going to request them for multiple to do them for single snapshots as well. That should cut down on the question of "when should I name the snapshot". Since it will be always it is easier to remember. Will update these.